tools/ThermoRawFileParser/thermo_converter.xml

```diff
@@ -85,7 +85,7 @@ ThermoRawFileParser.sh
         </param>
     </inputs>
     <outputs>
-        <data name="output" format="mzml" label="${tool.name} on ${on_string}">
+        <data name="output" format="mzml" label="${($input_file.name.rsplit('.',1)[0] if $input_file.name.lower().endswith('.raw') else $input_file.name)}.${('mgf' if $format_cond.output_format == '0' else 'mzml')}">
```

Comment:
I don't think we should do such magic. Its a Galaxy issue, that we should not fix per tool. The way we recommend today is using tags. Do tags help here? @subinamehta ?